src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedParameterSymbol.cs

```diff
@@ -169,6 +169,8 @@ public override string GetDocumentationCommentXml(CultureInfo preferredCulture =
             return _underlyingParameter.GetDocumentationCommentXml(preferredCulture, expandIncludes, cancellationToken);
         }
 
+        internal override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes => _underlyingParameter.InterpolatedStringHandlerArgumentIndexes;
```

Comment:
>internal override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes => _underlyingParameter.InterpolatedStringHandlerArgumentIndexes; [](http://example.com/codeflow?start=8&length=144)

I would prefer overriding this property only in leaf classes. Otherwise, it is hard to predict whether this implementation is appropriate for all derived types now and in the future.